components/proxy/src/test/kotlin/com/hotels/styx/routing/handlers/BackendServiceProxyTest.kt

```diff
@@ -0,0 +1,128 @@
+/*
+  Copyright (C) 2013-2019 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.routing.handlers
+
+import com.hotels.styx.Environment
+import com.hotels.styx.api.HttpResponseStatus.OK
+import com.hotels.styx.api.LiveHttpRequest
+import com.hotels.styx.api.LiveHttpResponse
+import com.hotels.styx.api.extension.Origin.newOriginBuilder
+import com.hotels.styx.api.extension.service.BackendService
+import com.hotels.styx.api.extension.service.spi.AbstractRegistry
+import com.hotels.styx.api.extension.service.spi.Registry
+import com.hotels.styx.api.extension.service.spi.Registry.ReloadResult.reloaded
+import com.hotels.styx.client.BackendServiceClient
+import com.hotels.styx.proxy.BackendServiceClientFactory
+import com.hotels.styx.routing.configBlock
+import com.hotels.styx.server.HttpInterceptorContext
+import io.kotlintest.shouldBe
+import io.kotlintest.shouldThrow
+import io.kotlintest.specs.StringSpec
+import reactor.core.publisher.Mono
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.CompletableFuture.completedFuture
+
+class BackendServiceProxyTest: StringSpec ({
```

Comment:
There's an space before the parenthesis, which is not consistent with the other tests.